### ISSUE TITLE
Can't load PDBs from Microsoft Symbol Server

### ISSUE THREAD

**Issue by jez9999** [aspnet/AspNetCore#13277]
I'm trying to debug ASP.NET Core by disabling Just My Code, enabling Source Link support, and selecting the Microsoft symbol servers as the symbol servers to load from.  However, when I try to set a breakpoint for `Invoke` (I'm trying to debug `StaticFileMiddleware.cs`), Visual Studio 2019 tells me the PDB isn't loaded ("Microsoft.AspNetCore.StaticFiles.pdb not loaded").  It looks like it couldn't be found in the symbol server.  Here's the symbol load information from the Modules window:

```
C:\Development\simplyclick-core\SimplyClick\bin\Debug\netcoreapp3.0\Microsoft.AspNetCore.StaticFiles.pdb: Cannot find or open the PDB file.
C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App\3.0.0-preview7.19365.7\Microsoft.AspNetCore.StaticFiles.pdb: Cannot find or open the PDB file.
C:\WINDOWS\Microsoft.AspNetCore.StaticFiles.pdb: Cannot find or open the PDB file.
C:\WINDOWS\symbols\dll\Microsoft.AspNetCore.StaticFiles.pdb: Cannot find or open the PDB file.
C:\WINDOWS\dll\Microsoft.AspNetCore.StaticFiles.pdb: Cannot find or open the PDB file.
C:\Users\Jez\AppData\Local\Temp\SymbolCache\Microsoft.AspNetCore.StaticFiles.pdb\ad52767ad76e4bf9899d8c6f52c07dffffffffff\Microsoft.AspNetCore.StaticFiles.pdb: Cannot find or open the PDB file.
C:\Users\Jez\AppData\Local\Temp\SymbolCache\MicrosoftPublicSymbols\Microsoft.AspNetCore.StaticFiles.pdb\ad52767ad76e4bf9899d8c6f52c07dffffffffff\Microsoft.AspNetCore.StaticFiles.pdb: Cannot find or open the PDB file.
C:\Users\Jez\AppData\Local\Temp\SymbolCache\Microsoft.AspNetCore.StaticFiles.pdb\ad52767ad76e4bf9899d8c6f52c07dff1\Microsoft.AspNetCore.StaticFiles.pdb: Cannot find or open the PDB file.
C:\Users\Jez\AppData\Local\Temp\SymbolCache\MicrosoftPublicSymbols\Microsoft.AspNetCore.StaticFiles.pdb\ad52767ad76e4bf9899d8c6f52c07dff1\Microsoft.AspNetCore.StaticFiles.pdb: Cannot find or open the PDB file.
C:\Development\simplyclick-core\Microsoft.AspNetCore.StaticFiles.pdb: Cannot find or open the PDB file.
SYMSRV:  BYINDEX: 0x1F9
         C:\Users\Jez\AppData\Local\Temp\SymbolCache*https://msdl.microsoft.com/download/symbols
         Microsoft.AspNetCore.StaticFiles.pdb
         AD52767AD76E4BF9899D8C6F52C07DFFffffffff

SYMSRV:  UNC: C:\Users\Jez\AppData\Local\Temp\SymbolCache\Microsoft.AspNetCore.StaticFiles.pdb\AD52767AD76E4BF9899D8C6F52C07DFFffffffff\Microsoft.AspNetCore.StaticFiles.pdb - path not found

SYMSRV:  UNC: C:\Users\Jez\AppData\Local\Temp\SymbolCache\Microsoft.AspNetCore.StaticFiles.pdb\AD52767AD76E4BF9899D8C6F52C07DFFffffffff\Microsoft.AspNetCore.StaticFiles.pd_ - path not found

SYMSRV:  UNC: C:\Users\Jez\AppData\Local\Temp\SymbolCache\Microsoft.AspNetCore.StaticFiles.pdb\AD52767AD76E4BF9899D8C6F52C07DFFffffffff\file.ptr - path not found

SYMSRV:  HTTPGET: /download/symbols/Microsoft.AspNetCore.StaticFiles.pdb/AD52767AD76E4BF9899D8C6F52C07DFFffffffff/Microsoft.AspNetCore.StaticFiles.pdb

SYMSRV:  HttpQueryInfo: 80190194 - HTTP_STATUS_NOT_FOUND

SYMSRV:  HTTPGET: /download/symbols/Microsoft.AspNetCore.StaticFiles.pdb/AD52767AD76E4BF9899D8C6F52C07DFFffffffff/Microsoft.AspNetCore.StaticFiles.pd_

SYMSRV:  HttpQueryInfo: 80190194 - HTTP_STATUS_NOT_FOUND

SYMSRV:  HTTPGET: /download/symbols/Microsoft.AspNetCore.StaticFiles.pdb/AD52767AD76E4BF9899D8C6F52C07DFFffffffff/file.ptr

SYMSRV:  HttpQueryInfo: 80190194 - HTTP_STATUS_NOT_FOUND

SYMSRV:  RESULT: 0x80190194

SYMSRV:  BYINDEX: 0x1FA
         C:\Users\Jez\AppData\Local\Temp\SymbolCache*https://msdl.microsoft.com/download/symbols
         Microsoft.AspNetCore.StaticFiles.pdb
         AD52767AD76E4BF9899D8C6F52C07DFF1

SYMSRV:  UNC: C:\Users\Jez\AppData\Local\Temp\SymbolCache\Microsoft.AspNetCore.StaticFiles.pdb\AD52767AD76E4BF9899D8C6F52C07DFF1\Microsoft.AspNetCore.StaticFiles.pdb - path not found

SYMSRV:  UNC: C:\Users\Jez\AppData\Local\Temp\SymbolCache\Microsoft.AspNetCore.StaticFiles.pdb\AD52767AD76E4BF9899D8C6F52C07DFF1\Microsoft.AspNetCore.StaticFiles.pd_ - path not found

SYMSRV:  UNC: C:\Users\Jez\AppData\Local\Temp\SymbolCache\Microsoft.AspNetCore.StaticFiles.pdb\AD52767AD76E4BF9899D8C6F52C07DFF1\file.ptr - path not found

SYMSRV:  HTTPGET: /download/symbols/Microsoft.AspNetCore.StaticFiles.pdb/AD52767AD76E4BF9899D8C6F52C07DFF1/Microsoft.AspNetCore.StaticFiles.pdb

SYMSRV:  HttpQueryInfo: 80190194 - HTTP_STATUS_NOT_FOUND

SYMSRV:  HTTPGET: /download/symbols/Microsoft.AspNetCore.StaticFiles.pdb/AD52767AD76E4BF9899D8C6F52C07DFF1/Microsoft.AspNetCore.StaticFiles.pd_

SYMSRV:  HttpQueryInfo: 80190194 - HTTP_STATUS_NOT_FOUND

SYMSRV:  HTTPGET: /download/symbols/Microsoft.AspNetCore.StaticFiles.pdb/AD52767AD76E4BF9899D8C6F52C07DFF1/file.ptr

SYMSRV:  HttpQueryInfo: 80190194 - HTTP_STATUS_NOT_FOUND

SYMSRV:  RESULT: 0x80190194

https://msdl.microsoft.com/download/symbols: Symbols not found on symbol server.
```

**Comment by Pilchie** [comment]
@dougbu any ideas?

**Comment by dougbu** [comment]
Could be one of two things:
1. Someone missed pushing symbols when we published v2.1.1 or v2.2.0 (the highest versions of Microsoft.AspNetCore.StaticFiles available on NuGet).
2. Pushes for our shared framework assemblies (like Microsoft.AspNetCore.StaticFiles.dll) weren't working for the specific release here.

@jez9999 which version of Microsoft.AspNetCore.StaticFiles.dll are you using? If it's for 3.0 preview 7, have you tried preview 8?

**Comment by pranavkm** [comment]
@jez9999 do you still see this problem with the 3.0 RTM release? I recall stepping in to the code of another binary in the Microsoft.AspNetCore.App shared fx recently, so it's fairly likely this is resolved.

If you're still seeing this with 3.0 RTM please comment here and we'll reopen to investigate.
